spec/02-integration/05-proxy/00-proxy_spec.lua

```diff
@@ -74,3 +75,43 @@ describe("Proxy interface listeners", function()
     end
   end)
 end)
+
+describe("#stream proxy interface listeners", function()
+  before_each(function()
+    helpers.get_db_utils()
+  end)
+
+  after_each(function()
+    helpers.stop_kong()
+  end)
+
+  it("disabled", function()
+    assert(helpers.start_kong({
+      stream_listen = "off",
+    }))
+    assert.equals(1, count_server_blocks(helpers.test_conf.nginx_kong_stream_conf))
+    assert.is_nil(get_listeners(helpers.test_conf.nginx_kong_stream_conf).kong)
+  end)
+
+  it("multiple", function()
+    assert(helpers.start_kong({
+      stream_listen = "127.0.0.1:9011, 127.0.0.1:9012",
+      plugins = "bundled",
```

Comment:
Maybe this option can be removed (not needed for the test, and is also the default).